draft-ietf-tls-tls13.md

```diff
@@ -2749,38 +2749,45 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+#### Age and Lifetime {#age-lifetime}
+
+When a client create a NewSessionTicket message, the ticket_age is
+calculated as follows:
+
+       ticket_age = client_create_psk_time - client_recv_nst_time
+
+If "ticket_age < ticket_lifetime * 1000" is satisfied, the client can
+use the ticket. obfuscated_ticket_age is calculated as follows:
+
+       obfuscated_ticket_age = ticket_age_add - ticket_age
```

Comment:
I thought it was

    obfuscated_ticket_age = ticket_age + ticket_age_add % 2^32

Whre '%' indicates modular arithmetic.